packages/ng/form-field/form-field.component.html

```diff
@@ -6,7 +6,9 @@
 	[class.u-mask]="hiddenLabel"
 	attr.aria-hidden="{{hiddenLabel}}"
 >
-	{{label}}<sup class="formLabel-required" aria-hidden="true" *ngIf="required">*</sup>
+	<ng-container *luPortal="label"></ng-container
+	><!--
+	--><sup class="formLabel-required" aria-hidden="true" *ngIf="required">*</sup>
```

Comment:
```suggestion
	<ng-container *luPortal="label"></ng-container
	><sup class="formLabel-required" aria-hidden="true" *ngIf="required">*</sup>
```

Reply:
The comment is useful here to prevent unwanted line break, see @vvalentin-lucca's comment above